Makefile

```diff
@@ -148,6 +148,7 @@ CCAN_HEADERS :=						\
 	$(CCANDIR)/ccan/array_size/array_size.h		\
 	$(CCANDIR)/ccan/asort/asort.h			\
 	$(CCANDIR)/ccan/autodata/autodata.h		\
+	$(CCANDIR)/ccan/base64/base64.h			\
```

Comment:
That *should* be internal to libwally; however we use static libwally, so it gets exposed.  We should be using shared libraries for libwally; both because that would properly hide its internal symbols, but also because ideally it would be shipped externally to our code and we'd use that by preference.

This is how we handle libsodium, for example.